pyccel/ast/basic.py

```diff
@@ -6,7 +6,7 @@
 class Basic(sp_Basic):
     """Basic class for Pyccel AST."""
     _fst = None
-
+    __slots__ = []
```

Comment:
Shouldn't this be `__slots__ = ['_fst']`?

Reply:
I am not sure if the `_fst = None` declaration needs declaring in the slots. It may be that it does need to be in the slots but that the current declaration needs to be removed. This needs some research